extensions/azurecore/src/azureResource/providers/database/databaseService.ts

```diff
@@ -42,14 +57,14 @@ export class AzureResourceDatabaseService implements IAzureResourceService<azure
 			if (serversForRg && !db.kind.endsWith('system') && svrIdRegExp.test(db.id)) {
```

Comment:
I assume you've tested this logic? The ID regex it's using seems to be pretty specific to SQL databases - do the Synapse ones also have the same ID format?

Reply:
Yes, they do apparently. (From the same SQL DB Query no less)